[Bug] error logs points to docker.yaml but..

### Description

i just noticed this after latest update in the logs


but all services are working fine and  my docker.yaml is properly set
```
pidocker:
   host: 192.168.1.3
   port: 2375

 my-docker:
   socket: /var/run/docker.sock
```
the pidocker is my other pc that is working fine, everything displays and works fine, is just the logs

Notice: i saw that logs are written only when i access Homepage interface

### Steps to reproduce

don't know

### homepage version

latest

### Installation method

Unraid

### Configuration

_No response_

### Container Logs

```
30/12/2022 17:43:24
[2022-12-30T17:43:24.409Z] error: (node:1) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
30/12/2022 17:43:24
(Use `node --trace-warnings ...` to show where the warning was created)
30/12/2022 17:43:26
[2022-12-30T17:43:26.340Z] error: Failed to discover services, please check docker.yaml for errors or remove example entries.
30/12/2022 17:43:26
[2022-12-30T17:43:26.340Z] error: Error: All docker servers failed to connect or returned no containers
```
however when i try this in the console `node --trace-warnings ...` i get this message 

```
/app # node --trace-warnings ...
node:internal/modules/cjs/loader:998
  throw err;
  ^

Error: Cannot find module '/app/...'
    at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Module._load (node:internal/modules/cjs/loader:841:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

### Browser Logs

_No response_

### Other

_No response_

### Before submitting, I have made sure to

- [X] Check [the documentation](https://gethomepage.dev/)
- [X] Follow [the troubleshooting guide](https://gethomepage.dev/en/more/troubleshooting/) (please include output above if applicable).
- [X] Search [existing issues](https://github.com/benphelps/homepage/search?q=&type=issues) and [discussions](https://github.com/benphelps/homepage/search?q=&type=discussions).

--- a/src/utils/config/api-response.js
+++ b/src/utils/config/api-response.js
@@ -50,6 +50,9 @@ export async function servicesResponse() {
 
   try {
     discoveredServices = cleanServiceGroups(await servicesFromDocker());
+    if (discoveredServices?.length === 0) {
+      console.debug("No containers were found with homepage labels.");
+    }
   } catch (e) {
     console.error("Failed to discover services, please check docker.yaml for errors or remove example entries.");
     if (e) console.error(e.toString());

--- a/src/utils/config/service-helpers.js
+++ b/src/utils/config/service-helpers.js
@@ -82,10 +82,6 @@ export async function servicesFromDocker() {
     })
   );
 
-  if (serviceServers.every(server => server.services.length === 0)) {
-    throw new Error('All docker servers failed to connect or returned no containers');
-  }
-
   const mappedServiceGroups = [];
 
   serviceServers.forEach((server) => {

